app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
@jywarren Ok for this friendly_id already has the solution. There will be no similar slugs for two nodes so a similar title will have a unique slug with a index appended to it like `NewYork-map--2` for a map that has a same name. So two maps with the same name won't conflict. If you say I can still modify the slug but then the url for maps would be like `/map/NewYork-map-07-22-2016/07-22-2016`